src/services/completions.ts

```diff
@@ -1590,7 +1590,8 @@ namespace ts.Completions {
                 if (m.kind !== SyntaxKind.PropertyAssignment &&
                     m.kind !== SyntaxKind.ShorthandPropertyAssignment &&
                     m.kind !== SyntaxKind.BindingElement &&
-                    m.kind !== SyntaxKind.MethodDeclaration) {
+                    m.kind !== SyntaxKind.MethodDeclaration &&
+                    m.kind !== SyntaxKind.GetAccessor) {

```

Comment:
I thought about it. Maybe we need more complex logic there. Let me explain it.
I think, that we have some cases. First one:
```
type Foo = {
  bar: string;
}
let x: Foo = {
  get bar() { return "" },
  /*$*/
}
```
In marker `$` we need completion about setter of `bar` (something like `set bar(bar) {}`). And vice versa for existing getter property.
And second one:
```
type Foo = {
  readonly bar: string;
}
let x: Foo = {
  get bar() { return ""; },
  /*$*/
}
```
In this case in marker `$` completions list must be empty, because `bar` is read only property. What do you think about it?

Unfortunately, for now TypeScript is allowing setter accessor for readonly property in object literal. Is it a bug?